src/components/comment.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import Edit from './edit';
+import Delete from './delete'
+
+const Comment = (props) => {
+  const [replyClicked, setReplyClicked] = useState(false);
+  const [reply, setReply] = useState('');
+  const [comments, setComments] = useState(props.comments);
+
+  const handleReplySubmit = (e) => {
+    e.preventDefault();
+    if (reply === '') {
+      return;
+    }
+
+    const replyData = {
+      id: 0,
```

Comment:
could you think about using this package to create a unique id? 

https://www.npmjs.com/package/uuid

Reply:
all done